src/super_gradients/common/environment/environment_config.py

```diff
@@ -12,4 +13,7 @@
 DDP_LOCAL_RANK = int(os.getenv("LOCAL_RANK", default=-1))
 EXTRA_ARGS = []
 
+# This needs to be run at the beginning because later on we drop --local_rank
+IS_TORCH_DISTRIBUTED_LAUNCH = any("--local_rank" in arg for arg in sys.argv)
```

Comment:
why do we need this now?

Reply:
I took it out in f434737b7551d1747cd3522d72f5dc1e2a03e6a4